tester_app/client/app/bundles/HelloWorld/startup/HelloWorldAppClient.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
@alexfedoseev should keep the Client in the file/component name when ONLY during client rendering?